send/ad_user_mu_ucn

```diff
@@ -191,14 +234,15 @@ sub process_update() {
 				unless ($response->is_error()) {
 					# SUCCESS
 					foreach my $log_message (@entry_changed) {
-						ldap_log($service_name, "Updated: " . $ad_entry->dn() . " | " . $log_message);
+						#ldap_log($service_name, "Updated: " . $ad_entry->dn() . " | " . $log_message);
```

Comment:
There should be a comment about this behavior. Because there are too many update operations, we don't want to spam the log if not necessary.